.github/workflows/backend.yml

```
@@ -61,7 +61,21 @@ jobs:
           distribution: "temurin"
           cache: maven
 
-      - name: Build
+      - name: Auto-fix Formatting (Maven Formatter)
+        run: mvn -B -ntp formatter:format
+
+      - name: Commit and Push Auto-fixes
+        if: github.event.pull_request.head.repo.full_name == github.repository
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)